docs-2.0/nebula-explorer/deploy-connect/ex-ug-connect.md

```diff
@@ -53,7 +53,7 @@ To connect Explorer to NebulaGraph, follow these steps:
 
     !!! note
 
-        - When NebulaGraph and Explorer are deployed on the same machine, you must enter the IP address of the machine, but not `127.0.0.1` or `localhost`, in the **Host** field.
+        - When NebulaGraph and Explorer are deployed on the same machine, you must enter the IP address of the machine, but not `127.0.0.1` or `localhost`.
```

Comment:
```suggestion
        - When NebulaGraph and Explorer are deployed on the same machine, you must enter the IP address of the machine, instead of `127.0.0.1` or `localhost`.
```